### ISSUE TITLE
[botskills] botskills connect should allow trailing backslash in --luisFolder

### ISSUE THREAD

**Issue by lauren-mills** [microsoft/botframework-solutions#1717]
## Project

**Name**: BotSkills CLI Tool

**Language**:  Typescript

## Description

When running the botskills connect tool with the --luisFolder parameter, if the path has a trailing backslash, the command fails with the following error

```
There was an error while connecting the Skill to the Assistant:
Error: An error ocurred while updating the Dispatch model:
Error: The execution of the ludown command failed with the following error:
Output folder ***\Deployment\Resources\LU\en" --out MySkill.luis does not exist
Stopping further processing.
```

## To Reproduce

1. Run botskills connect with --luisFolder path with trailing backslash

**Comment by darrenj** [comment]
@lauren-mills Could you try with the latest version of botskills, I think this was included in a fix earlier in the week which I've just published a new build for.

**Comment by lauren-mills** [comment]
Got it, just wasn't sure if it had been raised yet.

**Comment by Batta32** [comment]
Hi @lauren-mills,

As @darrenj said, we included the fix the last week creating the Pull Request #1665 related to this issue. We noticed different behaviors if the user uses _Command Prompt_ or _PowerShell_. For the first one, we created a new `Known Issues` explaining the situation. And for _PowerShell_, we fixed that updating the functionality of the `luisgen` execution.
However, we identified that in #1683 was reverted the changes.

We will create a Pull Request fixing this issue for `PowerShell`.
